### PR TITLE
[Upgrade] compatible to openai>=1.0.0

### PR DESCRIPTION
**Pull request by Luodian**
Since the package `openai` has big update in their Nov. 2023 release. The original code wont work with `openai>=1.0.0`, I made update accordingly and test in my local machine.

### CODE REVIEW

File: create_completion.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python3
 
 import openai
```
Comment (royerk) [inline create_completion.py:3]: Might as well drop this import (the `import openai` one).

**Review by royerk — COMMENTED** [review]
I was about to do the same, thanks for updating. Doing a review to notify the authors :)

File: create_completion.py (hunk 1 above)
Comment (Luodian) [reply create_completion.py]: Thanks! I did so.

**Comment by tom-doerr** [comment]
Thank you for fixing this @Luodian ! Thanks for the review @royerk

**Pull request merged**